README.md

```diff
@@ -78,15 +78,41 @@ XAMPP is a very easy to install Apache Distribution for Linux, Solaris, Windows
 
 This [video](https://youtu.be/Yzksa_WjnY0) walks you through the installation process for Windows but it should be similar for other OSs.
 
-### Config File
+### Docker
 
-DVWA ships with a dummy copy of its config file which you will need to copy into place and then make the appropriate changes. On Linux, assuming you are in the DVWA directory, this can be done as follows:
+It is possible to run DVWA with containers.
 
-`cp config/config.inc.php.dist config/config.inc.php`
+Prerequisites: Docker and Docker Compose.
 
-On Windows, this can be a bit harder if you are hiding file extensions, if you are unsure about this, this blog post explains more about it:
+- If you are using Docker Desktop, both of these should be already installed.
+- If you prefer Docker Engine on Linux, make sure to follow their [installation guide](https://docs.docker.com/engine/install/#server).
 
-[How to Make Windows Show File Extensions](https://www.howtogeek.com/205086/beginner-how-to-make-windows-show-file-extensions/)
+Then, to get started:
+
+1. Run `docker version` and `docker compose version` to see if you have Docker and Docker Compose properly installed. You should be able to see the version of Docker in the output.
+
+    ```text
+    >>> docker version
+    Client:
+     [...]
+     Version:           23.0.5
+     [...]
+
+    Server: Docker Desktop 4.19.0 (106363)
+     Engine:
+      [...]
+      Version:          23.0.5
+      [...]
+
+    >>> docker compose version
+    Docker Compose version v2.17.3
+    ```
+
+2. Clone or download this repository and extract (see [Download](download)).
+3. Open a terminal of your choice and change its working directory to `DVWA`.
+4. `docker compose up -d`.
+
+DVWA is now available at `http://localhost:4280`.
```

Comment:
You need to stress this alternate port number more, people will miss this or forget it when they come back to it. Some will also question why it isn't 80. Just ad a sentence or two to explain why.

Reply:
How do you stress in markdown? Or raw html has to be used here?